pkg/hubble

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Start daemon at boot time
+# Description:       Enable service provided by daemon.
+### END INIT INFO
+
+dir="/etc/hubble"
+cmd="hubble -d"
+user="root"
+
+name=`basename $0`
```

Comment:
Meh, I stole the template. I could run through and replace them with `$()` if it makes you feel better. :)